osu.Game.Rulesets.Osu/Skinning/Default/DefaultSmoke.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osu.Framework.Graphics;
+using osuTK;
+using osuTK.Graphics;
+
+namespace osu.Game.Rulesets.Osu.Skinning.Default
+{
+    public class DefaultSmoke : Smoke
```

Comment:
Bad visually? I separated default/legacy and gave default temporary effects assuming that default skin (or I guess triangles/argon?) would have special smoke effects, since they have flashy effects that are impossible to achieve with legacy skins.

Reply:
Bad visually, yeah... Maybe replace the default implementation with it and use that everywhere? Is the only suggestion I have for now. Curious what @peppy thinks of it.